db/migrate/20220324163932_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :posts_counter
```

Comment:
This migration should specify that these fields must not be null or empty strings.